### ISSUE TITLE
Content labeling error from Google Console

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2591]
Version: 2.2.1-alpha.1
System: Android and iOS
According to Google Console raport - This item may not have a label readable by screen readers

This is **EXTREMELY LOW PRIORITY** - we are not aiming to be fully functional with screen readers yet. This issue is described because it's pointed out by Google Console and if we start fixing accessibility we may start from it. 

**Issue**: for some reason "You can create a new community instead" are implemented in a way that suggest that each word (and space!) is a new sentence. Also "create a new community" should be either native link element or get an ARIA informing that it is.

Please see this recording with the sound on in order to understand an issue:
https://github.com/user-attachments/assets/d0c1bd36-8c5a-4a9a-b881-f84a2ec3ca32
